### PR TITLE
order tracking system

### PR DESCRIPTION
**Pull request by kamarbaraka**
implemented the order tracking system in the class OrderManagementService.

### CODE REVIEW

File: src/main/java/bussiness/services/management/OrderManagementService.java [hunk 1]
```diff
@@ -4,7 +4,316 @@
...
+    private final String PERSISTENCE_UNIT = "inventory_management";
+
+    /*declare pending orders list*/
+    private List<ItemOrder> pendingOrders;
+    /*declare dispatched orders list*/
+    private List<ItemOrder> dispatchedOrders;
+    /*declare completed orders lists*/
+    private List<ItemOrder> completedOrders;
+
+    public boolean addOrder(Address address, Payment payment, Customer customer, Bill bill, ItemOrder itemOrder){
```
Comment (develhope-team1) [inline src/main/java/bussiness/services/management/OrderManagementService.java:22]: looks good.

File: src/main/java/bussiness/services/management/OrderManagementService.java [hunk 2]
```diff
@@ -4,7 +4,316 @@
...
+            manager.persist(customer);
+            manager.persist(bill);
+            manager.persist(itemOrder);
+
+            /*commit the transaction*/
+            transaction.commit();
+
+            return true;
+        }
+        finally {
```
Comment (develhope-team1) [inline src/main/java/bussiness/services/management/OrderManagementService.java:53]: good for closing

File: src/main/java/bussiness/services/management/OrderManagementService.java [hunk 3]
```diff
@@ -4,7 +4,316 @@
...
+        return dispatchedOrders;
+    }
+
+    public List<ItemOrder> getCompletedOrders() {
+        return completedOrders;
+    }
+
+    /**
+     * testing */
+    public static void main(String[] args) {
```
Comment (develhope-team1) [inline src/main/java/bussiness/services/management/OrderManagementService.java:304]: works fine

**Review by develhope-team1 — APPROVED** [review]

File: src/main/java/bussiness/services/management/OrderManagementService.java (hunk 1 above)
Comment (kamarbaraka) [reply src/main/java/bussiness/services/management/OrderManagementService.java:22]: thanks

**Comment by WILHEMS** [comment]
Great

**Pull request merged**